src/rsa/verification.rs

```diff
@@ -14,75 +14,14 @@
 
 //! Verification of RSA signatures.
 
-use super::{parse_public_key, RsaParameters, N, PUBLIC_KEY_PUBLIC_MODULUS_MAX_LEN};
+use super::{parse_public_key, public, RsaParameters, N, PUBLIC_KEY_PUBLIC_MODULUS_MAX_LEN};
 use crate::{
     arithmetic::{bigint, montgomery::Unencoded},
     bits, cpu, digest, error,
     limb::LIMB_BYTES,
     sealed, signature,
 };
 
-#[derive(Debug)]
-pub struct Key {
-    pub(super) n: bigint::Modulus<N>,
-    pub(super) e: bigint::PublicExponent,
-    pub(super) n_bits: bits::BitLength,
-}
-
-impl Key {
-    pub fn from_modulus_and_exponent(
```

Comment:
Note that when I review a change like this that purports to just move code around, I use a diff tool to explicitly compare the old and new file(s). So, I would open verification.rs as "old" and key.rs as "new" in my diff tool and compare them.